### DISCUSSION TITLE
How to set the log level of pybullet?

### DISCUSSION THREAD

**Discussion by borninfreedom** [bulletphysics/bullet3#4039] [Ideas]
There is a lot of information I don't want. How to block them and only show error messages?
![image](https://user-images.githubusercontent.com/48934506/140936862-47f52cfa-032d-47c9-9605-1eda4d9533fb.png)

**Comment by erwincoumans** [comment]
There is no such thing, best to fix the URDF file, read the feedback. Good idea though, moving to discussion.

**Comment by rohit-kumar-j** [comment]
@erwincoumans perhaps if there was a -v (verbose) flag or input arg to the p.loadURDF function?

**Comment by elevenjiang1** [comment]
Any solution? I meet the same problem, meanwhile, I still want to remove the output of above:
![image-20230722135743629](https://github.com/bulletphysics/bullet3/assets/43636049/5161705e-9f7b-4cd5-98de-d55decea7cfc)
